packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -198,6 +200,32 @@ const genFullBackCompatConfig = (): CompatConfig[] => {
 	return _configList;
 };
 
+export function isCompatVersionBelowMinVersion(
+	minVersion: string,
+	allVersions: string[],
+	config: CompatConfig,
+) {
+	let lowerVersion: string | number;
+	if (config.kind === CompatKind.CrossVersion) {
+		const compatV = allVersions.indexOf(config.compatVersion as string);
```

Comment:
Not sure I understand why allVersions is needed. Why not semver compare the 2 versions here?

Reply:
I just push a commit with that approach